Source/DafnyRuntime/DafnyRuntimeRust/src/lib.rs

```diff
@@ -3898,51 +3539,51 @@ pub mod rcmut {
     }
     /// Retrieve the inner Rc as a reference.
     pub unsafe fn from<T>(value: Box<T>) -> RcMut<T> {
-        mem::transmute(Rc::new(*value))
+        unsafe { mem::transmute(Rc::new(*value)) }
```

Comment:
The function is declared unsafe, why do you still add these unsafe blocks?

Reply:
Because that behavior is deprecated.
A function is unsafe if it makes assumptions about the input that the compiler cannot enforce.
A statement is unsafe if it does one of the bad things, like dereferencing a pointer.
Those two things are unrelated.
In the bad old days, someone got them confused, and unsafe functions could use unsafe statements willy nilly.
In the near future, that is going away, and unsafe statements will need to be marked as unsafe, no matter what kind of function they're in.

I turned on the unsafe_op_in_unsafe_fn warning, and so was forced to clean these up.